webpack/assets/javascripts/react_app/API.js

```diff
@@ -1,6 +1,18 @@
 import $ from 'jquery';
+import axios from 'axios';
+
+const onSuccess = response => response.data;
+const onError = error => Promise.reject(error);
```

Comment:
whats the rational for us to recreate all exceptions? e.g. I would expect a library to have predefined good exceptions for regular http failure codes vs we have to handle them ourselves?